app/controllers/subscribers_controller.rb

```diff
@@ -0,0 +1,41 @@
+class SubscribersController < ApplicationController
+
+    def new
```

Comment:
why do you have the `new` route defined with no logic in it?

Reply:
We have a before_action:
`before_action :get_article, only: [:show, :edit, :update]`

To perform this:
```
def get_article
        @article = Article.find(params[:id])
    end
```

This was to make the code more DRY

Reply:
I though we needed for the rout and view.
Removed